src/Extend/Notification.php

```diff
@@ -70,7 +70,13 @@ public function extend(Container $container, Extension $extension = null)
         });
 
         $container->extend('flarum.notification.drivers', function ($existingDrivers) {
-            return array_merge($existingDrivers, $this->drivers[0]);
+            $drivers = [];
+
+            array_walk($this->drivers, function ($driverData, $driverName) use (&$drivers) {
```

Comment:
Yeah let's change this to Arr::pluck for simplicity, If that wasn't an option, I'd prefer to use `array_map` instead of walk to avoid unnecessary mutations: gotta cling to the little bits of functional programming we can! :D